pilot/cmd/pilot-agent/status/ready/probe.go

```diff
@@ -72,5 +72,9 @@ func (p *Probe) checkServerState() error {
 		return fmt.Errorf("server is not live, current state is: %v", admin.ServerInfo_State(*state).String())
 	}
 
+	if ws != 1 {
```

Comment:
is this always 0 or 1? if so why not make this a bool here.

Reply:
Yes. It is always zero or 1. I can make it bool.